services/app-web/src/components/SectionHeader/SectionHeader.tsx

```diff
@@ -5,9 +5,9 @@ import classNames from 'classnames'
 
 export type SectionHeaderProps = {
     header: string
-    navigateTo?: string
     children?: React.ReactNode
-    subHeaderComponent?: React.ReactNode
+    editNavigateTo?: string // Controls appearance of edit link to the right of main heading - should be pathname substring for use with NavLink
+    subHeaderComponent?: React.ReactNode // Controls appearance of additional component below main heading
```

Comment:
thanks for adding this comment and replacing the `alignTop` class to be more semantic!